app/controllers/users/sessions_controller.rb

```diff
@@ -1,13 +1,22 @@
 class Users::SessionsController < Devise::SessionsController
   respond_to :json
+  skip_before_action :require_no_authentication
 
-  private
 
-  def respond_with(_resource, _opts = {})
+  # def respond_with(_resource, _opts = {})
```

Comment:
delete code, not comment

Reply:
Removed in commit "[commit-29] Remove unnecessary code"